pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -146,6 +146,14 @@
     )
     private String advertisedAddress;
 
+    //
+    @FieldContext(category=CATEGORY_SERVER, doc = "")
```

Comment:
Can you add documentation for these two settings?

Also, can you add these settings to `broker.conf` file?

Reply:
I have add the configuration doc here